util/cgroup/cgroup_cpu_test.go

```diff
@@ -42,9 +42,13 @@ func TestGetCgroupCPU(t *testing.T) {
 		}()
 	}
 	cpu, err := GetCgroupCPU()
-	require.NoError(t, err)
-	require.NotZero(t, cpu.Period)
-	require.Less(t, int64(1), cpu.Period)
+	if err == errNoCPUControllerDetected {
+		require.False(t, InContainer(), "Please check linux version. This is related to cgroup compatibility.")
```

Comment:
better to figure out in comments which version of Linux is supported. Like `> 4.x` or `>= 5.x` etc.

Reply:
Thx, I have updated it.